app/src/main/java/com/islaharper/dawnofandroid/navigation/NavGraph.kt

```diff
@@ -0,0 +1,19 @@
+package com.islaharper.dawnofandroid.navigation
+
+import androidx.compose.runtime.Composable
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import com.islaharper.dawnofandroid.presentation.screens.splash.SplashScreen
+
+@Composable
+fun SetupNavGraph(navController: NavHostController) {
```

Comment:
This function is a composable but it is called `SetupNavGraph` (a verb). To be consistent with the other Composables, I reckon it should have a name that forms a noun:

https://github.com/androidx/androidx/blob/androidx-main/compose/docs/compose-api-guidelines.md#naming-unit-composable-functions-as-entities

Maybe you can have a look at some other projects to see how they have named this part of the code?

https://github.com/android/nowinandroid

Reply:
Resolved in latest commit.